tests/tools/oesign/test-enclave/enclave/enc.c

```diff
@@ -52,3 +53,61 @@ bool is_test_signed()
 
     return is_test_signed;
 }
+
+oe_result_t check_kss_extended_ids(
+    oe_uuid_t* family_id,
+    oe_uuid_t* ext_product_id)
+{
+    oe_result_t result = OE_UNEXPECTED;
+    size_t report_size = OE_MAX_REPORT_SIZE;
+    uint8_t* remote_report = NULL;
+    oe_report_header_t* header = NULL;
+    sgx_quote_t* quote = NULL;
+    uint64_t quote_size = 0;
+    char isvid_hex[33];
```

Comment:
Consider defining a constant or #define for magic number 33 and reusing the constant throughout this code, e.g.
```c
/* Null-terminated hex string buffer size with 2 char per byte */
const size_t OE_KSS_ID_HEX_BUFFER_SIZE = sizeof(oe_uuid_t) * 2 + 1;
```